packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianInteraction.selectors.ts

```diff
@@ -120,28 +113,25 @@ export const selectorChartsInteractionXAxisValue = createSelector(
 );
 
 export const selectorChartsInteractionYAxisValue = createSelector(
-  [
-    selectorChartsInteractionPointerY,
-    selectorChartYAxis,
-    selectorChartsInteractionYAxisIndex,
-    optionalGetAxisId,
-  ],
-  (y, yAxes, yIndex, id) => {
+  [selectorChartsInteractionPointerY, selectorChartYAxis, selectorChartsInteractionYAxisIndex],
+  (y, yAxes, yIndex, id?: AxisId) => {
     if (y === null || yAxes.axisIds.length === 0) {
       return null;
     }
     return valueGetter(y, yAxes, yIndex, id);
   },
 );
 
+const noop: AxisItemIdentifier[] = [];
```

Comment:
Might be more descriptive:

```suggestion
const EMPTY_ARRAY: AxisItemIdentifier[] = [];
```